classes/Optimization/File.php

```diff
@@ -534,8 +535,8 @@ public function optimize( $args = [] ) {
 			$args['convert'] = '';
 		}
 
-		if ( 'webp' === $args['convert'] ) {
-			$destination_path = $this->get_path_to_webp();
+		if ( 'webp' === $args['convert'] || 'avif' === $args['convert'] ) {
```

Comment:
```suggestion
		$formats = [
		'webp',
		'avif',
		];
		
		if ( in_array( $args['convert'], $formats, true ) ) {
```
Something like this avoids to do multiple conditions, and will be easy to extend for the future

Reply:
Done